commercial/public/facebook_ia_remote.html

```diff
@@ -0,0 +1,126 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="robots" content="noindex">
+    <script>
+    (function (window) {
+        var guardian = window.guardian = parseParameters(window.location);
+        var googletag = window.googletag = { cmd: [] };
+        window.Krux || ((window.Krux=function(){Krux.q.push(arguments)}).q=[]);
+        window.config = {
+            pageId: guardian.url,
+            edition: toTitleCase(guardian.edition),
+            seriesId: guardian.se,
+            contentType: toTitleCase(guardian.ct),
+            keywordIds: guardian.keywordIds,
+            keywords: guardian.k.split(',').map(toTitleCase),
+            authorIds: guardian.authorIds,
+            blogIds: guardian.bl,
+            referrer: 'facebook-ia',
+            section: guardian.section
+        };
+
+        function parseParameters(location) {
+            return location.hash.substr(1)
+            .split('&')
+            .reduce(function (result, arg_i) {
+                var arg = arg_i.split("=");
+                var argName = decodeURIComponent(arg[0]);
+                var argValue = decodeURIComponent(arg[1]);
+
+                if (result[argName] !== undefined) {
+                    result[argName] += ',' + argValue;
+                } else {
+                    result[argName] = argValue;
+                }
+
+                return result;
+            }, {});
+        }
+
+        function toTitleCase(str) {
+            return str.replace(/\w\S*/g, function(txt) {
+                return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+            });
+        }
+    }(window))
+    </script>
+    <script async class="kxct" data-id="J_RxuxWJ" data-timing="async" data-version="1.9" src="https://cdn.krxd.net/controltag?confid=J_RxuxWJ"></script>
+    <script async src="https://www.googletagservices.com/tag/js/gpt.js"></script>
+</head>
+<body style="margin:0">
+    <div id="ad-slot-1"></div>
+    <script>
+    (function (window, localStorage, guardian, undefined) {
```

Comment:
ha! good catch. It is a trick to make `undefined` a local reference when using an IIFE. But I moved code around and this is not needed anymore, not there. Thanks :-)